services/backend/src/persistence/tropicalDryForest.js

```diff
@@ -1,40 +1,4 @@
 module.exports = (db, { geoTropicalDryForestDetails, geoHFTropicalDryForest }) => ({
-  /**
-   * Get the area inside the given environmental authority
-   *
-   * @param {String} eaId environmental authority id
-   * @param {Number} year optional year to filter data, 2012 by default
-   */
-  findAreaByEA: (eaId, year = 2012) =>
-    geoTropicalDryForestDetails
-      .query()
-      .where({ id_ea: eaId, year_cover: year })
-      .select(db.raw('coalesce(SUM(area_ha), 0) as area')),
-
-  /**
-   * Get the area inside the given basin subzone
-   *
-   * @param {String} subzoneId basin subzone id
-   * @param {Number} year optional year to filter data, 2012 by default
-   */
-  findAreaBySubzone: (subzoneId, year = 2012) =>
-    geoTropicalDryForestDetails
-      .query()
-      .where({ id_subzone: subzoneId, year_cover: year })
-      .select(db.raw('coalesce(SUM(area_ha), 0) as area')),
-
-  /**
-   * Get the area inside the given state
-   *
-   * @param {String} stateId state id
-   * @param {Number} year optional year to filter data, 2012 by default
-   */
-  findAreaByState: (stateId, year = 2012) =>
-    geoTropicalDryForestDetails
-      .query()
-      .where({ id_state: stateId, year_cover: year })
-      .select(db.raw('coalesce(SUM(area_ha), 0) as area')),
-
```

Comment:
The same as these functions in paramo, and also for wetland

Reply:
Done!